lib/probability/core_ext/float.rb

```diff
@@ -0,0 +1,21 @@
+require 'securerandom'
+
+class Float
+  #Extention of Float class to include probability function.
+  #TODO: Determine maximum precision and edge cases, design class around language limitations?
+  #
+  # 1.35.in(9.99): true ~13.51% of the time
+  # and so on...
+
+  def in(number, &block)
```

Comment:
I see *a lot* of duplication between `Float#in` and `Interger#in`. It might be useful to extract this code and generalize it in a module.